### ISSUE TITLE
Tool invoke error

### ISSUE THREAD

**Issue by ptrprkrz** [langgenius/dify#4325]
### Self Checks

- [X] This is only for bug report, if you would like to ask a quesion, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### Dify version

0.6.7

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

Search web with duckduckgo tool

### ✔️ Expected Behavior

Retrieved information from a duckduckgo search

### ❌ Actual Behavior

REQUEST TO DUCKDUCKGO_SEARCH
{"duckduckgo_search": {"query": "openai news tomorrow"}}
RESPONSE FROM DUCKDUCKGO_SEARCH
{"duckduckgo_search": "tool invoke error: https://links.duckduckgo.com/d.js?q=openai+news+tomorrow&kl=wt-wt&l=wt-wt&p=&s=0&df=y&vqd=4-81530519592241638441885684796925095938&ex=-1 202"}

**Comment by patryk20120** [comment]
After upgrading I have `202 Ratelimit` which is strange because there is only one request being sent

**Comment by saddestboy** [comment]
I met the same problem telling me 202.
```json
{"duckduckgo_search": "tool invoke error: https://links.duckduckgo.com/d.js?q=origin+of+the+letter+A&kl=wt-wt&l=wt-wt&p=&s=0&df=y&vqd=4-138105515496736000197279088754094215604&ex=-1 202"}
```

**Comment by kota-iizuka** [comment]
This problem seems to be caused by a lower version of `duckduckgo-search==5.2.2`. A minimal example that reproduces the same error is as follows:

```
>>> from duckduckgo_search import DDGS
>>> DDGS().text("llm")
duckduckgo_search.exceptions.RatelimitException: https://links.duckduckgo.com/d.js?q=llm&kl=wt-wt&l=wt-wt&p=&s=0&df=&vqd=4-329897...&ex=-1 202
```

This bug can be fixed by updating `duckduckgo-search==6.1.0`.

```
>>> DDGS().text("llm")
[{'title': 'Large language model - Wikipedia', 'href': 'https://en.wikipedia.org/wiki/Large_language_model', 'body': 'A large language model (LLM) is a computational model ...
```

**Comment by kiron111** [comment]
they do it intensionally, force the customer to buy their online service....
DAMN....even the searxng docker installation.... they add a -rm command on the documetation, so it will remove everytime after you reboot....
and the google search....actually almost a free service provide by google....they give you only 100 free search by month (not even by day)
